subsys/bluetooth/host/hci_core.c

```diff
@@ -7778,6 +7925,12 @@ static int le_ext_adv_param_set(struct bt_le_ext_adv *adv,
 	atomic_set_bit_to(adv->flags, BT_ADV_USE_IDENTITY,
 			  param->options & BT_LE_ADV_OPT_USE_IDENTITY);
 
+	atomic_set_bit_to(adv->flags, BT_ADV_LEGACY,
```

Comment:
Rebase and use the new `BT_ADV_EXT_ADV` flag: https://github.com/zephyrproject-rtos/zephyr/pull/27052/files#diff-902572dfd5d05bc64ec198af0ad3162fR93

Reply:
Updated :)